src/main/kotlin/racingcar/controller/Controller.kt

```diff
@@ -28,10 +30,18 @@ class Controller {
 
     private fun inputPlayTime(): Int {
         return try {
-            InputView.inputPlayTime()
+            PlayTime(InputView.inputPlayTime()).value
         } catch (e: Exception) {
             InputView.printError(e.message!!)
             inputPlayTime()
         }
     }
+
+    private fun startRace(cars: Cars) {
+        cars.play(RandomMovingStrategy())
+        cars.cars.forEach {
+            ResultView.printResult(it.name.value, it.position.value)
+        }
```

Comment:
car 내부 상태 값을 외부로 노출하는것 보단 필요한 결과를 전달 받는 구조는 어떨까요 ?

```suggestion
        val racingResults = cars.play(RandomMovingStrategy())
        racingResults.forEach {
            ResultView.printResult(it.name.value, it.position.value)
        }
```